components/StateHandlers/InitialStateHandler.tsx

```diff
@@ -25,6 +25,10 @@ const getSchemedURLFromUniversalURL = (url: string) => {
   return schemedURL;
 };
 
+const isDevelopmentClientURL = (url: string) => {
```

Comment:
Want to make a ticket to fix this stuff up a bit more?

I think there's some issues with the current integration

Reply:
Oh! Like this is causing other issues? Or the reloading is still happening anyway?